### PR TITLE
[[ Bug 20480 ]] shell command should separate return value and the result

### PR DESCRIPTION
**Pull request by trevordevore**

### CODE REVIEW

File: docs/dictionary/function/shell.lcdoc [hunk 1]
```diff
@@ -31,15 +31,16 @@ get shell("say" && "hello world") -- Will say the phrase on MacOS
...
 <commandLine> generates. (On <Unix|Unix systems>, the <stdout> and
 <stderr> are combined to create the <return value>.) The current
-<handler> pauses until the <shell> returns its result. If the command
+<handler> pauses until the <shell> <function> returns its result. If the command
 was successful but did not return anything, the <shell> <function>
-returns empty. The result is set to the shell command's exit code.
+returns empty. 
+
+The result:
+The shell command's exit code.
```
Comment (montegoulding) [inline docs/dictionary/function/shell.lcdoc]: @trevordevore It returns empty if the exit code was 0

Comment (trevordevore) [reply docs/dictionary/function/shell.lcdoc]: @montegoulding Fixed.

**Comment by montegoulding** [comment]
@livecode-vulcan review ok 7883b43

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok 7883b43e6ffda1418dd377911a698237c98f902d

**Comment by livecode-vulcan** [comment]
😎 test success 7883b43e6ffda1418dd377911a698237c98f902d

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/138)
* try-community-armv7-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-sdk26_ndk16r15/builds/130)
* try-community-arm64-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-sdk26_ndk16r15/builds/130)
* try-community-x86-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-sdk26_ndk16r15/builds/130)
* try-community-x86_64-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-sdk26_ndk16r15/builds/130)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/1652)
* try-community-universal-ios-iphoneos12.0: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos12.0/builds/34)
* try-community-universal-ios-iphonesimulator12.0: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator12.0/builds/34)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1682)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1675)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1672)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1708)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/1649)

**Pull request merged**